core/lib/libtomcrypt/src/misc/mem_neq.c

```diff
@@ -74,6 +74,11 @@ int mem_neq(const void *a, const void *b, size_t len)
       ++pb;
    }
 
+   ret |= ret >> 4;
+   ret |= ret >> 2;
+   ret |= ret >> 1;
+   ret &= 1;
+
```

Comment:
This was a comment raised in Libtomcrypt (https://github.com/libtom/libtomcrypt/issues/74). Tom St Denis answered the following

*"!!ret" could in theory have branches in it. If you want a fixed time implementation the folding is the way to go*